docs/guides/chip_tool_guide.md

```diff
@@ -257,14 +257,15 @@ Alternatively, you can also use a QR code payload.
 
 ##### Commissioning with setup PIN code
 
-To discover devices and try to pair with the first discovered device using the
-provided setup code, use the following command pattern:
+To discover devices and try to pair with the first available discovered device
+using the provided setup code, use the following command pattern:
```

Comment:
The "first available" bit does not match the actual behavior (which is described correctly in lines 267-268).